### PR TITLE
SC-5265 - lernstore-arrow-back-window

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
changed reuirements, right now it should be full size window with arrow to close it

## Links to Tickets or other pull requests

- https://github.com/schul-cloud/nuxt-client/pull/1115
- https://ticketsystem.schul-cloud.org/browse/SC-5265

### CODE REVIEW

File: static/scripts/topicEdit.js [hunk 1]
```diff
@@ -600,7 +600,7 @@ class TopicResources extends TopicBlock {
         if(!resource) {
             let isCourseGroupTopic = $contentBlocksContainer.data('iscoursegroup') !== undefined;
             // open content search popup
-            const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, "content-search", "toolbar=no, location=no, directories=no, width=800,height=600,status=no,scrollbars=yes,resizable=yes");
+			const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, '_self', "content-search", "toolbar=no, location=no, directories=no, width=800,height=600,status=no,scrollbars=yes,resizable=yes");
```
Comment (mwolowyk-cap) [inline static/scripts/topicEdit.js:603]: Please rename resourcePopup since it is not popup anymore

File: static/scripts/topicEdit.js [hunk 2]
```diff
@@ -600,7 +600,9 @@ class TopicResources extends TopicBlock {
         if(!resource) {
             let isCourseGroupTopic = $contentBlocksContainer.data('iscoursegroup') !== undefined;
             // open content search popup
-            const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, "content-search", "toolbar=no, location=no, directories=no, width=800,height=600,status=no,scrollbars=yes,resizable=yes");
+            const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, "content-search",
+				`width=1920, height=1080, fullscreen=yes, toolbar=no, location=no, directories=no, status=no, scrollbars=yes, resizable=yes`);
```
Comment (BlackGhostHD) [inline static/scripts/topicEdit.js:604]: Does this also work with other screen resolutions, e.g. Smartphones?

Comment (ABialoskorski) [reply static/scripts/topicEdit.js:604]: Mona told me that when it was previously 800x600 on mobile it was just opening a fullscreen window
so I assume this is the same behaviour

Comment (mwolowyk-cap) [reply static/scripts/topicEdit.js:604]: If it is bigger than screen it opens in full screen

**Pull request merged**